packages/preview/exzellenz-tum-thesis/0.1.0/template/Chapter_Introduction.typ

```diff
@@ -0,0 +1,84 @@
+#import "utils.typ": todo
+
+= Introduction <chapter_introduction>
+
+== Motivation
+
+#todo[Write the thesis]
+
+Because I have to @cite_todo. This is a abbreviation @oidc explained in the glossary.
+
+#lorem(100)
+
+#lorem(100)
+
+
+== Research Gap
+
+As shown in @figure_dummy, this is wonderful. Typst documentation #footnote(link("https://typst.app/docs")) or that way #footnote[Google einfach]. Look at @eq_einstein or write it $E = M C^2$ inline.
+
+#figure(
+    grid(
+        columns: 3,     // 2 means 2 auto-sized columns
+        row-gutter: 2mm,    // space between columns
+        [#image("dummy_image.svg", width: 70%)],
+        [#image("dummy_image.svg", width: 70%)],
+        [#image("dummy_image.svg", width: 70%)],
+        [#image("dummy_image.svg", width: 70%)],
+        [#image("dummy_image.svg", width: 70%)],
+        [#image("dummy_image.svg", width: 70%)],
+    ),
+  caption: [
+    #lorem(10)
+  ]
+) <figure_dummy>
+
+#lorem(40)
+
+$
+E = M C^2
+$ <eq_einstein>
+
+#lorem(60)
+
+
+#figure(
+[#set text(size: 0.8em, hyphenate: false, weight: "medium"); #set par(justify: false)
+  #let gut(it) = [#text(fill: rgb("006600"))[#it]];
+  #let mittel(it) = [#text(fill: rgb("333300"))[#it]];
+  #let schlecht(it) = [#text(fill: rgb("660000"))[#it]];
+  #table(
+  columns: (12.2em, 1fr, 1fr, 1fr, 1fr),
+  rows: 5em,
+  fill: (col, row) => if row == 0 or col == 0 { luma(240) } else { none },
+  inset: 5pt,
+  align: horizon,
+  [], [*Method A*], [*Method B*], [*Method C*], [*Method D*],
+  [_property_], [#schlecht[No] #footnote[#lorem(10)] <fntable1>], [#gut[Yes] #footnote[#lorem(10)] <fntable2>], [#gut[Yes] #footnote(<fntable2>)], [#gut[Yes] #footnote(<fntable2>)],
+  [_memory_], [#schlecht[No] #footnote(<fntable1>)], [#gut[Yes] #footnote(<fntable2>)], [#gut[Yes] #footnote(<fntable2>)], [#gut[Yes] #footnote(<fntable2>)],
+  [_complexity_], [#gut[Low] #footnote[#lorem(10)] <fntable4>], [#gut[Low] #footnote(<fntable4>)], [#schlecht[High] #footnote(<fntable6>)], [#schlecht[High] #footnote(<fntable6>)],
+  [_runtime_], [#gut[Low] #footnote(<fntable4>)], [#gut[Low] #footnote(<fntable4>)], [#schlecht[High] #footnote[#lorem(10)] <fntable6>], [#schlecht[High] #footnote(<fntable6>)],
+)],
```

Comment:
Since you only use function calls in here, you can make the brackets curly and omit the hashtags. I'd also recheck indentation, Typst mostly uses the style

```typ
#figure(
  {
    // Block indented with two spaces against braces
  },
  caption: lorem(30), // Use trailing comma. Alternatively, with content block:
  caption: [
    Table of Comparative Categories
  ],
)
```

As we saw with our default templates, you'd be surprised at how little people change both the source code and output styling, so this is our chance to make the style good!

Reply:
I think 
```
caption: [
    #lorem(30)
]
```
is actually better, because this way people can just replace `#lorem(30)` with the actual input.

Reply:
Change the indentation and the use of {} instead of []